File: R-Language/pattern.r
for (i in 1:6){
    for(j in 1:i){
        cat(" * ")
    }
    message(" ")
}
message("Print a Pattern in another Form")
for(i in 1:6){
    for(j in i:6){
        cat(" * ")
    }
    message(" ")
}
#while loop
x<-1
while(x<=10){
    cat(" ",x)
    x = x+1
}
#repeated loop
y<-1
repeat{
    message(y," ")
    y = y+1
    if(y==5)
    break
}

File: C#/Carbon-Programing-Language/Competitve Question/pattern.cpp
#include<bits/stdc++.h>
using namespace std;
class Pattern{
    protected:
    int n;
    //cin>>n;
};
class Loop:public Pattern{
    public:
    void pattern(int a){
        n = a;
        for(int i =1; i<=n; i++){
            for(int j=1; j<=i; j++){
                if(j==i-j){
                    cout<<" ";
                }
                else
                cout<<" * ";
            }
            cout<<endl;
        }
    }
};
int main(){
    Loop l1;
    l1.pattern(6);
}